src/main/py/peer/inventory.py

```diff
@@ -46,7 +50,11 @@ def add_pending_trade(self, trade_key, trade_offer):
         - trade_key (bytes): The key of the artwork to add to the inventory.
         - trade_offer (OfferAnnouncement | OfferResponse): The trade offer to add to the inventory.
         """
-        self.artworks_pending_trade.add(trade_offer.artwork_ledger_key)
+
+        if isinstance(trade_offer, OfferResponse):
```

Comment:
With the rename in `OfferResponse` this block now equates to the same thing as the `elif` block. So, you can revert this change and keep the original statement.